packages/paste-website/src/components/site-wrapper/SiteBody.tsx

```diff
@@ -35,10 +37,32 @@ const StyledSiteBody = styled.div<StyledSiteBodyProps>`
   }
 `;
 
+const BLMAlert: React.FC = () => (
+  <Box
+    css={{backgroundColor: '#000', color: '#fff', borderColor: '#000'}}
+    borderBottomWidth="borderWidth20"
+    borderBottomStyle="solid"
+    paddingLeft="space70"
+    paddingRight="space70"
+    paddingTop="space40"
+    paddingBottom="space30"
+    role="status"
+    display="flex"
+    justifyContent="center"
+  >
+    <strong>Black Lives Matter</strong>
+    &nbsp;&nbsp;
+    <Anchor css={{color: '#fff'}} href="https://twitter.com/twilio/status/1266444750065934337">
```

Comment:
Just ned to fix up the hover state
![image](https://user-images.githubusercontent.com/368249/83433982-81983000-a3ee-11ea-959c-fc8d1ac9911b.png)